packages/s2-core/__tests__/unit/interaction/root-spec.ts

```diff
@@ -20,7 +20,7 @@ import {
   BrushSelection,
   RowColumnResize,
   DataCellMultiSelection,
-  ShiftMultiSelection,
+  RangeMultiSelection,
```

Comment:
```suggestion
  RangeSelection,
```
`Range`已经说明是`Multi`了